domains/www.dear.json

```
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "kirokir",
+    "email": "itsmelorika@gmail.com"
+  },
+  "records": {
+    "CNAME": "dearnco-django.onrender.com"
+  },
+  "proxied": true
```

you can't proxied onrender
```suggestion
  }
```

---

@STICKnoLOGIC Thank you so much for the feedback and for catching that! I’ve removed the "proxied": true setting as suggested since it’s hosted on Render. Please let me know if there’s anything else I can improve. Grateful for your support! 😊